velox/common/caching/CachedFactory.h

```diff
@@ -155,15 +69,14 @@ class CachedFactory {
   // repeatedly, handing off the results to one thread at a time until the
   // all pending requests are satisfied or a cache insert succeeds. This
   // will probably mess with your memory model, so really try to avoid it.
-  CachedPtr<Key, Value, Comparator, Hash> generate(const Key& key);
+  std::pair<bool, Value> generate(const Key& key);
```

Comment:
Can we update the documentation here, to explain the return types

Reply:
+1
```
The function sets true in the returned pair if the generated value is a cache hit or false if it is a new insert.
```